### ISSUE TITLE
How many nodes/machines are used for training

### ISSUE THREAD

**Issue by flintstuff** [wyhlovecpp/GPT-Image-Edit#1]
Thank you so much for selflessly sharing the code and dataset, really appreciated. And I'm curious how many machines have been used during training, 1.5M samples is truly a lot, I wonder how many machines and how long does it take for each stage? Thank you!

**Comment by wyhlovecpp** [comment]
Thanks for your interest! We trained on a single node with 8×A100 GPUs. Full hyperparameters are in our config:
[stage2_training_config](https://github.com/wyhlovecpp/GPT-Image-Edit/blob/7c1fc653f0d5e5a984b5073b14a2d4d52a93909f/scripts/denoiser/flux_qwen2p5vl_7b_vlm_stage2_1024.yaml)

Training lasted for over a week and covered nearly one epoch. If more resources become available, we’ll scale up like UniWorld-V1 — larger batch size, learning rate, and more epochs — to further explore FluxKontext's potential.
